### PR TITLE
branch: building-html-form

### PR DESCRIPTION
**Pull request by bigmosi**
Add form to the portfolio page.
Add form style to index.css.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,204 +1,234 @@
...
-      <img src="assets/images/Rectangle.png" class="rect70" alt="">
+      <img src="assets/images/Rectangle.png" class="rect70" alt="" />
       <p class="last-text">ruby</p>
+    </div>
+    <button class="button-1" type="submit">See Project</button>
 
-  </div>
-  <button class="button-1" type="submit">See Project</button>
+    <div class="multi4">
+      <h1 class="contact-info">CONTACT US</h1>
```
Comment (elmar8287) [inline index.html:190]: - [ ] Please, consider to use correct fonts for "Contact us" header. It should be `Roboto `, as mentioned in the Figma template
Screen Shot 2022-02-22 at 16 12 59

- [ ] Please, consider using the `h1` header tag one time in your code. All other headers should be `hx`, aka h2, h3, ...

File: index.html [hunk 2]
```diff
@@ -1,204 +1,234 @@
...
-  <div class="down-image">
-      <hr class="hr">
+      <label for="email"></label>
+      <input type="email" name="email" id="email" placeholder="Enter an email" required class="text-input" />
+
+      <label for="textarea"></label>
+      <textarea cols="30" rows="3" id="textarea" placeholder="Enter text here" class="text-area" required maxlength="500"></textarea>
+
+      <label id="submit"></label>
+      <input type="submit" value="See Project" class="Button-1" />
```
Comment (elmar8287) [inline index.html:208]: - [ ] Please, consider the button style, as mentioned in the Figma template (hover, pressed, etc..):

Screen Shot 2022-02-22 at 16 18 51

**Review by elmar8287 — CHANGES_REQUESTED** [review]

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @bigmosi.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the comments under the review._

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 3]
```diff
@@ -369,3 +366,44 @@ ul {
...
+}
+
+.Button-1 {
+  border: none;
+  background-color: #7d73fc;
+  padding: 10px;
+  margin-top: 30px;
+  transition: ease 0.1s;
+  color: #fff;
+}
```
Comment (Wes-Isaac) [inline index.css:400]: - [ ] As the previous reviewer pointed out please consider the button style, as mentioned in the Figma template **(hover, pressed, etc..)**

image

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @bigmosi,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by thecodechaser — APPROVED** [review]
## Hi @bigmosi 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**